README.md

```diff
@@ -104,7 +104,8 @@ erDiagram
         description string
         created_at datetime
         updated_at datetime
-        deadline datetime
+        start_at datetime
```

Comment:
Does _start_at_ not the same as _created_at_? If no, then, please, rename _start_at_ to _started_at_ and _end_at_ to _finished_at_. And, i think we should have property _deadline_ to know about theretical ending of the sprint.

Reply:
created_at and updated_at is about working with entity. Those are service fields. But start_at and end_at are theoretical starting and ending dates.

Reply:
I talking about that _sprint_ can be finished after it's _deadline_. If you don't wanna rename them - that's ok, but don't remove _deadline_, please.

Reply:
end _at is a deadline. We don't have fields for real ending date. That's why field names are in present but not in past tense.